salt/modules/win_servermanager.py

```diff
@@ -57,11 +58,29 @@ def _pshell_json(cmd, cwd=None):
     if 'convertto-json' not in cmd.lower():
         cmd = '{0} | ConvertTo-Json'.format(cmd)
     log.debug('PowerShell: %s', cmd)
-    ret = __salt__['cmd.shell'](cmd, shell='powershell', cwd=cwd)
+    ret = __salt__['cmd.run_all'](cmd, shell='powershell', cwd=cwd)
+
+    if 'pid' in ret:
+        del ret['pid']
```

Comment:
Can you explain why using a `del` here is needed? I would imagine a `pop` would be fine instead, but I might be missing something here.

Reply:
Ah, right. Thanks.